decentraland/unity-explorer
# chore: add different send rates for moving and still player

## What does this PR change?

Decreased message send rate for network movement:
10 HZ (10 msg/sec) -when moving 
1 HZ (1 msg/sec)- when still


## How to test the changes?

1. Launch the explorer with your test buddy 
2. .Verify that movement of your friend as in prod
